Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -135,6 +135,7 @@ open class ALKConversationViewModel: NSObject, Localizable {
     private var groupMembers: Set<ALContact>?
     private var awsEncryptionPrefix = "AWS-ENCRYPTED"
     private var botDelayTime = 0
+    private var configuration : ALKConfiguration?
```

Comment:
Let's keep the configuration in controller only. From VC, we should check and call an appropriate VM function based on the config value.